pkg/format/format.go

```diff
@@ -24,13 +25,14 @@ import (
 	"github.com/kanisterio/kanister/pkg/log"
 )
 
+var regex *regexp.Regexp
+
+func init() {
+	regex = regexp.MustCompile("[\r\n]")
+}
```

Comment:
nit: does the following work?

```suggestion
var regex = regexp.MustCompile("[\r\n]")
```

Reply:
@julio-lopez let me check

Reply:
@julio-lopez you are right. It's working. I have tested changes end to end and updated gist with new logs.